sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/TxOutputsView.kt

```diff
@@ -17,47 +17,48 @@ internal class SentNoteTable(
         private val ORDER_BY = String.format(
             Locale.ROOT,
             "%s ASC", // $NON-NLS
-            SentNoteTableDefinition.COLUMN_INTEGER_ID
+            TxOutputsViewDefinition.COLUMN_INTEGER_TRANSACTION_ID
         )
 
-        private val PROJECTION_ID = arrayOf(SentNoteTableDefinition.COLUMN_INTEGER_ID)
+        private val PROJECTION_ID = arrayOf(TxOutputsViewDefinition.COLUMN_INTEGER_TRANSACTION_ID)
 
         private val PROJECTION_RECIPIENT = arrayOf(
-            SentNoteTableDefinition.COLUMN_STRING_TO_ADDRESS,
-            SentNoteTableDefinition.COLUMN_INTEGER_TO_ACCOUNT
+            TxOutputsViewDefinition.COLUMN_STRING_TO_ADDRESS,
+            TxOutputsViewDefinition.COLUMN_INTEGER_TO_ACCOUNT
         )
 
-        private val SELECT_BY_TRANSACTION_ID = String.format(
+        private val SELECT_BY_TRANSACTION_ID_AND_NOT_CHANGE = String.format(
             Locale.ROOT,
-            "%s = ?", // $NON-NLS
-            SentNoteTableDefinition.COLUMN_INTEGER_TRANSACTION_ID
+            "%s = ? AND %s == 0", // $NON-NLS
+            TxOutputsViewDefinition.COLUMN_INTEGER_TRANSACTION_ID,
+            TxOutputsViewDefinition.COLUMN_INTEGER_IS_CHANGE
```

Comment:
I *think* this is generally okay, since shielding transactions should treat all the shielded value as "change".